src/components/PupilComponents/QuizResultOrder/QuizResultOrder.tsx

```diff
@@ -23,7 +23,7 @@ export const QuizResultOrder = ({
   pupilAnswers,
 }: QuizResultOrderProps) => {
   const resultItems = pupilAnswers.map((pupilAnswer, index) => {
-    const answer = answers[pupilAnswer];
+    const answer = answers[pupilAnswer - 1];
```

Comment:
I'm going to leave it but I don't really agree with the data transformation happening inside the component. This means that the interfaces are inconsistent between parallel components and will mean that the next person who comes across this has to scratch their head for a while. Can you at least make sure there is inline documentation in each component file about how the data is transformed ?